tests/python_package_test/test_dask.py

```diff
@@ -133,6 +133,11 @@ def test_classifier(output, centers, client, listen_port):
         centers=centers
     )
 
+    X_1, y_1, w_1, dX_1, dy_1, dw_1 = _create_data(
+        objective='classification',
+        output='array'
+    )
```

Comment:
Why was this necessary? You should just use the `dask_classifier` defined below this. With this change, you'd only be doing the local predict on arrays each time, but we want to test on all of DataFrame, Array, and sparse matrix.

Reply:
Initially, I tried to reference the prediction inside `test_classifier` with `p1_local=dask_classifier.to_local().predict(dX)` for `p1` of `test_classifier_local_predict`, but this line gave me two errors saying `ValueError: Expected 2D array, got scalar array instead` while running the tests.
I thought it was because of the difference we have in the `output` params in 
```
X, y, w, dX, dy, dw = _create_data(
        objective='classification',
        output=output,
        centers=centers
    )
```
and 
```
X, y, w, dX, dy, dw = _create_data(
        objective='classification',
        output='array'
    )
```
in `test_classifier` and `test_classifier_local_predict` accordingly. Is there any way to resolve it?

Reply:
`dask_classifier.to_local()` produces a `lightgbm.sklearn.LGBMClassifier`. Pass it `X` (a non-Dask data structure) instead of `dX`(a Dask data structure) and it should work without needing to have this extra `_create_data()`

Reply:
Thanks, it worked. I also renamed the variables. you can tell me if I need any other changes?